rke/templates/calico.go

```diff
@@ -505,6 +505,11 @@ metadata:
 spec:
   hostNetwork: true
   restartPolicy: OnFailure
+  tolerations:
```

Comment:
This change could cause plugins redeploy on pre-existing clusters. Can you check if the spec update followed by pre-existing cluster modification  - any parameter that would trigger rke cluster reconcile - result in addon redeploy? @orangedeng

Reply:
I had the same question :) Discussed with @galal-hussein and yeah, the addons upgrades would happen